src/EnergyPlus/Fans.cc

```diff
@@ -1604,14 +1498,15 @@ void SizeFan(EnergyPlusData &state, int const FanNum)
             ShowContinueError(state,
                               format("Invalid Fan Curve Name = \"{}\" does not cover ",
                                      state.dataFaultsMgr->FaultsFouledAirFilters(jFault_AirFilter).FaultyAirFilterFanCurve));
-            ShowContinueError(state, format("the operational point of Fan {}", Fan(FanNum).FanName));
+            ShowContinueError(state, format("the operational point of Fan {}", fan.FanName));
             ShowFatalError(
                 state,
                 format("SizeFan: Invalid FaultModel:Fouling:AirFilter={}", state.dataFaultsMgr->FaultsFouledAirFilters(jFault_AirFilter).Name));
         }
     }
 
-    if (++NumFansSized == state.dataFans->NumFans) FanNumericFields.deallocate(); // remove temporary array for field names at end of sizing
+    if (++NumFansSized == state.dataFans->NumFans)
+        state.dataFans->FanNumericFields.deallocate(); // remove temporary array for field names at end of sizing
```

Comment:
I doubt this is still working as intended with the declaration at the top as `int NumFansSized = 0;`.

Reply:
@mjwitte this is an example of what I think was a SAVE'd variable that is now local. Not sure how this could work as intended. NumFansSized is declared up at 1209.

Reply:
> Using SmallOffice_CentralDOAS, NumFansSized is always 0 at a breakpoint at line 1483 when NumFans = 4. So this is not working as intended. Is there a way to save this variable other than using state?

Not that I know of. But for this particular array that appears to be used to retrieve a single field name based on hard-wired field numbers, it should go away.

Reply:
Module local variables are global variables and we are not using them anymore. It's `state` or nothing.